stl/inc/utility

```diff
@@ -802,6 +818,26 @@ _NODISCARD constexpr underlying_type_t<_Ty> to_underlying(_Ty _Value) noexcept {
     _CSTD abort(); // likely to be called in debug mode, but can't be relied upon - already entered the UB territory
 #endif // _DEBUG
 }
+
+template <class _Ty, class _Uty>
+_NODISCARD constexpr auto&& forward_like(_Uty&& _Ux) noexcept {
+    static_assert(_Can_reference<_Ty>, "std::forward_like's first template argument must be a referenceable type.");
+
+    using _UnrefU = remove_reference_t<_Uty>;
+    if constexpr (is_lvalue_reference_v<_Ty>) {

```

Comment:
I believe it's a more obvious copying of the standard to put the `is_const_v` on the outside of the if block.